subprojects/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -35,6 +35,23 @@ The previous step will help you identify potential problems by issuing deprecati
 . Run `gradle wrapper --gradle-version {gradleVersion}` to update the project to {gradleVersion}.
 . Try to run the project and debug any errors using the <<troubleshooting.adoc#troubleshooting, Troubleshooting Guide>>.
 
+[[changes_8.5]]
+== Upgrading from 8.4 and earlier
+
+=== Potential breaking changes
+
+==== Changes to `JavaBasePlugin` conventions
```

Comment:
Like we talked about... these should be user facing.  We don't need to include details about internals if they don't impact most users